src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -300,7 +300,7 @@
 
     <!-- Get the previously-built-source-built package information from the manifest from that build. -->
     <ItemGroup>
-      <_PrebuiltSourceBuiltAssetManifests Include="$(PrebuiltSourceBuiltPackagesPath)VerticalManifest.xml" />
+      <_PrebuiltSourceBuiltAssetManifests Include="$(PrebuiltSourceBuiltPackagesPath)*.xml" />
```

Comment:
This is temporary right and was fixed in https://github.com/dotnet/sdk/pull/45554?  Assuming so, when will this get reverted and what is tracking that work?

Reply:
It can be reverted upon the next rebootstrap. The artifacts we target now don't have the fixed output. I'll log an issue to follow up.

Reply:
https://github.com/dotnet/source-build/issues/4808